server/planning/types/planning_types.py

```diff
@@ -0,0 +1,48 @@
+from datetime import datetime
+from pydantic import Field
+from typing import Annotated, Any
+
+from superdesk.core.resources import ResourceModel, fields
+from superdesk.utc import utcnow
+from superdesk.core.resources.fields import ObjectId
+from superdesk.core.resources.validators import validate_iunique_value_async, validate_data_relation_async
+
+
+class PlanningTypesResourceModel(ResourceModel):
+    # The name identifies the form in the UI to which the type relates
+    name: Annotated[fields.Keyword, validate_iunique_value_async("planning_types", "name")]
+
+    editor: dict[str, Any] = Field(
+        default_factory=dict,
+        description="Editor controls which fields are visible in the UI",
```

Comment:
:+1:  Nice use of description on the field, this will be handy in the future (when we auto-generate API docs)